winml/lib/Api/LearningModel.h

```diff
@@ -120,13 +116,10 @@ struct LearningModel : LearningModelT<LearningModel> {
       winml::ILearningModelFeatureDescriptor descriptor,
       bool& use_fp16);
 
-  void
-  OverrideShapeInferenceMethods();
```

Comment:
>OverrideShapeInferenceMethods [](start = 2, length = 29)

where did this method go?

Reply:
IWinMLAdapter::OverrideSchemaInferenceFunctions()

---
In reply to: [347041742](https://github.com/microsoft/onnxruntime/pull/2406#discussion_r347041742) [](ancestors = 347041742)